### ISSUE TITLE
Migrate from pgapp to sumo_db

### ISSUE THREAD

**Issue by unbalancedparentheses** [lambdaclass/holiday_pinger#142]
The database code is getting more complex. We should migrate to a layer like sumo db to manage the db entities. The only issue is that we should update sumo db to use pgapp as the new postgresql driver.

**Comment by unbalancedparentheses** [comment]
@igaray do you think you can do this?

**Comment by igaray** [comment]
Yes, I took a look at this and started. It seems it won't be as straightforward as possible for two reasons: 
- there is no easy way of making it a gradual transition.
- HP uses sql in ways that escape the repository model

This is more or less the roadmap I have:
- Add `sumo_db` and `sumo_db_pgsql` dependencies.
- Translate the `db_user`, `db_channel`, `db_reminder`, and `db_holiday` modules to `sumo_doc` behaviors. Just add the schema definitions, try to match the current table definitions as closely as possible and see whether sumo can at least create and query model entities. 
- Create repo modules for the aforementioned entities.
- Create a repo extension for holiday ping to house the complex queries.
